src/modules/meteor/index.js

```diff
@@ -185,7 +185,8 @@ export function start(api) {
   list.executeScript('Start Meteor', {
     script: resolve(__dirname, 'assets/meteor-start.sh'),
     vars: {
-      appName: config.name
+      appName: config.name,
+      nginxClientUploadLimit: config.nginxClientUploadLimit || '10M'
```

Comment:
I think you want to add this variable for `start.sh` instead, which is on line 127.